### PR TITLE
Tcs 111 updating config table

### PR DESCRIPTION
**Pull request by AdamBrown2**
Combined all previous csv's into one segmenting.csv in TorQ/config
Updated TorQ/code/segmentedtickerplant/datastripe.q to use the one csv
Unit test currently being worked on

### CODE REVIEW

File: code/segmentedtickerplant/datastripe.q [hunk 1]
```diff
@@ -1,24 +1,20 @@
 //configload loads configurable .csv files into memory so relevant filters can be applied to data.
-//loads segmenting.csv which defines how many segments there will be.
-//loads filtermap.csv which defines what filters should be applied to each segment.
 
 //.ds.segmentconfig/segmentfiltermap are variables which can take different csv files. These files can be chosen on in process.csv.
 //error trap these .ds variables incase this file is loaded stand alone.
 
-.ds.segmentconfig:@[value;`.ds.segmentconfig;`segmenting.csv];
-.ds.filtermap:@[value;`.ds.filtermap;`filtermap.csv];
+.ds.segmentconfig:@[value;`.ds.segmentconfig;`segmenting1.csv];
```
Comment (jonathonmcmurray) [inline code/segmentedtickerplant/datastripe.q:6]: should this be `segmenting.csv` or is `segmenting1.csv` intentional?

File: code/segmentedtickerplant/datastripe.q [hunk 2]
```diff
@@ -30,29 +26,33 @@ initdatastripe:{
...
      filter:segmentfilter[tbl;segid];
-     if[filter~"";
-          .lg.e[`sub;m:"Incorrect pairing of table ",string[tbl]," and segmentID ",string[segid]," not found in .stpps.segmentconfig"];
-          :();
-     ];
-     
-     .ps.subtablefiltered[string[tbl];filter;""]
+     $[tbl in .stpps.fullsubscriptions;[.stpps.suball[tbl]];
+     [$[tbl in .stpps.ignoredtables;
+                    [.lg.o[`sub;m:"Table ",string[tbl]," is in ignoredtables.csv and will not be subscribed to"];
```
Comment (jonathonmcmurray) [inline code/segmentedtickerplant/datastripe.q:49]: I think this log message probably needs updated?

File: code/segmentedtickerplant/datastripe.q [hunk 3]
```diff
@@ -30,29 +26,33 @@ initdatastripe:{
...
+                    :();];
+       [$[filter~"";             
+                    [.lg.e[`sub;m:"Incorrect pairing of table ",string[tbl]," and segmentID ",string[segid]," not found in .stpps.segmentconfig"];
+                    :(tbl;`err`msg!(`segmentid;m));]
+                    ;
+                    [.ps.subtablefiltered[string[tbl];filter;""]]]]]]]
```
Comment (jonathonmcmurray) [inline code/segmentedtickerplant/datastripe.q:55]: I think this nested conditional is a bit confusing and hard to follow - could it be either re-written as separate `if` statements, or re-formatting in some way?

e.g. I think something like the following would have the same functionality but is easier to follow IMO using early exits so that each subsequent if statement only gets executed if the previous ones were false:

```suggestion
     if[tbl in .stpps.fullsubscriptions;:.stpps.suball[tbl]];
     if[tbl in .stpps.ignoredtables;
        .lg.o[`sub;m:"Table ",string[tbl]," is in ignoredtables.csv and will not be subscribed to"];
        :();
     ];
     if[filter~"";          
        .lg.e[`sub;m:"Incorrect pairing of table ",string[tbl]," and segmentID ",string[segid]," not found in .stpps.segmentconfig"];
        :(tbl;`err`msg!(`segmentid;m));
     ];
     .ps.subtablefiltered[string[tbl];filter;""]
```

File: code/segmentedtickerplant/datastripe.q (hunk 2 above)
Comment (AdamBrown2) [reply code/segmentedtickerplant/datastripe.q:49]: Yes i missed that thanks

File: code/segmentedtickerplant/datastripe.q (hunk 3 above)
Comment (AdamBrown2) [reply code/segmentedtickerplant/datastripe.q]: Did not know it could be done like that and it all works Thanks
